layer/trace_layer.cpp

```diff
@@ -51,7 +52,16 @@ const VkLayerProperties kLayerProps = {
         GFXRECON_PROJECT_VERSION_DESIGNATION
 };
 
-const std::vector<VkExtensionProperties> kDeviceExtensionProps = { VkExtensionProperties{ "VK_EXT_tooling_info", 1 } };
+const std::vector<VkExtensionProperties> kDeviceExtensionProps = {
```

Comment:
> Sorry, if this is const, why was it ever a vector?

For that you'd have to ask the original author.

A helpful suggestion might be "if we're not taking advantage of any methods of `std::vector`, can this be a const array?"  The reply might be, "you'd have to change the references to `kDeviceExtensionProps` below.  That may not be a very big change but would require more code changes more than just adding values to this vector."

My thought would be, is `vector` so distasteful because of performance or size concerns that we need to change it?  I think the answer would be "no" because there's only one of these, and you know I'm a fan of not changing things from their existing style unless there's a compelling reason.  But if your thought makes sense to David then I'm also okay with him changing it.